src/git_changelog/cli.py

```diff
@@ -259,6 +287,76 @@ def _unreleased(versions: list[Version], last_release: str) -> list[Version]:
     return versions
 
 
+def read_config(
+    config_file: str | list[str] | None = DEFAULT_CONFIG_FILES,
+) -> dict:
+    """Find config files and initialize settings with the one of highest priority.
+
+    Arguments:
+        config_file: A path or list of paths to configuration file(s); or ``None`` to
+            disable config file settings. Default: ``pyproject.toml`` and ``.git-changelogrc``
+            at the current working directory.
+
+    Returns:
+        A settings dictionary. Default settings if no config file is found or ``config_file``
+        is ``None``.
+
+    """
+    project_config = DEFAULT_SETTINGS.copy()
+    if config_file is None:  # Unset config file
+        return project_config
+
+    config_file = [config_file] if isinstance(config_file, str) else config_file
+
+    for filename in config_file:
+        _path = Path(filename)
+
+        if not _path.exists():
+            continue
+
+        new_settings = toml.load(_path)
+        if _path.name == "pyproject.toml":
+            new_settings = new_settings.get("tool", {}).get("git-changelog", {})
+
+            if not new_settings:  # Likely, pyproject.toml did not have a git-changelog section
+                continue
+
+        # Settings can have hyphens like in the CLI
+        new_settings = {
+            key.replace("-", "_"): value for key, value in new_settings.items()
+        }
+
+        # Massage found values to meet expectations
+        # Parse sections
+        if "sections" in new_settings and new_settings["sections"] is not None:
+            sections = new_settings["sections"]
+            if isinstance(sections, str):
+                sections = [s.strip() for s in sections.split(",")]
```

Comment:
It means we accept both strings and lists, right? Maybe the docs should show both uses then?